src/Tooltip/mobile-view.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react'
+import ReactMarkdown from 'react-markdown'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+class MobileView extends Component {
```

Comment:
@iAdramelk Alex, you have mentioned that it's better to use pure react component, could you please share the link again?

Reply:
@shcheklein This component is a pure component. But it is not inherited from one.

Reply:
Hi. I think we can leave <Tooltip> as is here. They are used inside of <Markdown> and it is already a PureComponent, so there will not be unnecessary rerenders anyway.